languages/emacs-lisp/README.md

```diff
@@ -0,0 +1,54 @@
+# Emacs Lisp
+
+Welcome to the workspace for Exercism v3's Emacs Lisp track!
+
+This area will contain everything needed to launch the Emacs Lisp track, including:
+
+- The new exercises and `config.json` file.
+- Reference documentation that help explain Emacs Lisp concepts for which no appropriate online document could be found.
+- Emacs Lisp-specific documentation for contributors.
+
+## Preparation Status
+
+Before we publicize requesting contribution for this language, the following steps should be done.
+
+- [ ] Have a kick-off discussion between track maintainers
+- [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
+- [ ] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md))
```

Comment:
There's an extra `)` at the end of this line. You may wish to leave it there as a "low hanging" PR for a new maintainer, or tidy it now. :)

(I accidently added it everywhere - sorry!)